starter-code/src/functions-and-arrays.js

```diff
@@ -60,6 +137,20 @@ var wordsFind = [
   'disobedience'
 ];
 
+function doesWordExist(arr, str) {
```

Comment:
No es necesario incluir el else if, podemos solucionarlo por ejemplo de esta manera:

function doesWordExist(arr, str) {
  if (arr.length === 0) {
      return false
  }
  for (let i = 0; i < arr.length; i++) {
    if (arr[i] === str || arr.includes(str)) {
      return true
  }
      return false
  }
}

asi tenemos una solución con menos condicionales, variables y mas limpia